tokenizer/gpt2tok.cc

```diff
@@ -97,22 +96,17 @@ using json = nlohmann::json;
 class VocabData {
  public:
   VocabData()
-    : unk_id_(-1) {
+      : unk_id_(-1) {
   }
 
   struct BpeNode {
     int id;
     int value;
   };
 
-  void Load(const char* p_vocab_file, const char* p_bpe_file, const char* unk_token, const char* special_tokens) {
-    std::ifstream json_stream(p_vocab_file);

```

Comment:
Did you guys verify the performance of  the method Load, c++ stream is not very performant in some case.